### ISSUE TITLE
How can i put a dense layer in the last layer for classification?

### ISSUE THREAD

**Issue by matgreco** [sufeidechabei/graphnets_gcn#2]
Hi,
 Thank's for your job. Looks interesting.

How can i put a dense layer as last layer in the network, for classifications tasks.

Thank's.

**Comment by sufeidechabei** [comment]
You can add nn.Linear(hidden_units, num_classes) as the output layer.

**Comment by sufeidechabei** [comment]
You can change the final layer function. (change the snt.linear(num_classes) to whatever you need.)

**Comment by sufeidechabei** [comment]
Now, you can use our new project. The accuracy is even better than the result in the original paper https://github.com/dmlc/dgl/tree/master/examples/pytorch/gcn
